pkg/util/pod.go

```diff
@@ -52,7 +46,6 @@ func GetRunningPod(ctx context.Context, kube client.Client) (*v1.Pod, error) {
 	}
 
 	name := types.NamespacedName{Name: podName, Namespace: podNamespace}
-	log.V(logging.Debug).Info("getting pod", "name", name)
```

Comment:
What's the reason to remove this?

Reply:
You know to be honest I think I thought this was a reconciler implementation and removed it while removing all the package level loggers.

That said, I do feel strongly that there should be no package level loggers in crossplane-runtime. We could refactor this function to take a logger as an argument, but I question the value of doing so. My preference would be to refactor this function such that it returns error messages with useful context (i.e. uses `errors.Wrap`) and then move the logging up to whatever code is invoking it.

Reply:
I'm going to avoid making any improvements to this function because it is part of a deprecated package. It should be moved out of crossplane-runtime and into whatever package uses it (the stack manager?), at which point it would make sense to refactor it for better logging.